packages/app-project/stores/Project.spec.js

```diff
@@ -0,0 +1,73 @@
+import Project from './Project'
+import Store from './Store'
+import stubPanoptesJs from '../test/stubPanoptesJs'
+import projectFixture, { initialState } from '../test/fixtures/project'
+
+let rootStore
+let projectStore
+
+describe.only('Stores > Project', function () {
+  it('should exist', function () {
+    rootStore = Store.create()
+    projectStore = rootStore.project
+    projectStore.should.not.be.undefined
+  })
+
+  describe('default model properties', function () {
+    before(function () {
+      rootStore = Store.create()
+      projectStore = rootStore.project
+    })
+
+    it('should have a displayName property', function () {
+      projectStore.displayName.should.equal('')
+    })
+
+    it('should have an error property', function () {
+      expect(projectStore.error).to.equal(null)
+    })
+
+    it('should have an id property', function () {
+      projectStore.id.should.equal('')
+    })
+
+    it('should have a state property', function () {
+      projectStore.state.should.equal('initialised')
```

Comment:
Same comment about importing these constants from the asyncStates file.

Reply:
👍